flex-templates/python/regional_dlp_transform/bigquery_dlp_bigquery.py

```diff
@@ -266,6 +313,81 @@ def process(self, element, **kwargs):
         yield operation.item
 
 
+@experimental()
+class MaskDetectedDetails(PTransform):
+
+    def __init__(
+            self,
+            project=None,
+            location="global",
+            template_name=None,
+            deidentification_config=None,
+            timeout=None):
+
+        self.config = {}
+        self.project = project
+        self.timeout = timeout
+        self.location = location
+
+        if template_name is not None:
+            self.config['deidentify_template_name'] = template_name
+        else:
+            self.config['deidentify_config'] = deidentification_config
```

Comment:
We might be able to remove the notion of the deid_config variable.....is the definition of the actual de-id config (rather than the template) possible? I thought we limited inputs to only template since defining the json for the config would be complex.  

Also, consider fixing the original code, which looks like it also left this check/dead variable in place

Reply:
The variables have been removed. They really weren't being used.